### ISSUE TITLE
County Info Page

### ISSUE THREAD

**Issue by nikobooth** [sfbrigade/stop-covid19-sfbayarea#236]
Reworking the current "news" page into a County Info page with selector to toggle between counties.

Mockup can be found on the Figma file on the 'wireframes for development' page: https://www.figma.com/file/iUS5LdAnuTKVei88QwPPCH/Bay-Area-Pandemic-Dashboard-PanDa-by-Bay-Area-Brigades?node-id=2161%3A8680

![image](https://user-images.githubusercontent.com/63436818/86291600-0fa93700-bba4-11ea-9fa4-f7166aac773e.png)
*note that while this image shows the new header image applied, it is not a prerequisite to implementing these changes

Interaction: County drop down selector shows all available counties

Navigation: Sidebar navigation similar to the FAQ page allows visitors to skip to specific sections

Content: First container has the 5 most recent news items for the selected county. Where available, the content summary should be pulled in below the link title. If visitors click on 'see all news items' they are taken to a separate archive page that displays all news items for all counties (similar to current layout) will need to include a 'back' link at the top of this page to take users back to county info:
![image](https://user-images.githubusercontent.com/63436818/86290428-5007b580-bba2-11ea-93b4-7659b253f4b3.png)

Static content and font/formatting notations will be provided in the google sheet: https://docs.google.com/spreadsheets/d/1_wBXS62S5oBQrwetGc8_-dFvDjEmNqzqHwUeP-DzkYs/edit#gid=974216782

Each county will have a specified sheet named <COUNTYNAME> - Info. This content will need to be maintained, I'll organize members of the brigade to help with this. Please advise on the best/easiest workflow for submitting changes and new content.

There are currently 7 counties with info ready to go:
San Francisco
Alameda
San Mateo
Santa Clara
Napa
Sonoma
Solano

Santa Cruz also has content but we should decide if we want to include Info for a county that we're not offering statistics for. Contra Costa and Marin counties will be added in the coming week.

Once these changes have gone through, the landing page should have this text adjusted:
![image](https://user-images.githubusercontent.com/63436818/86291090-59dde880-bba3-11ea-8639-832fea02da70.png)

new text will say
"County Information"
"What's happening in my county?"

**Comment by frhino** [comment]
Is this a good first issue?

**Comment by nikobooth** [comment]
> Is this a good first issue?

I think it would be! What do you think @thuongho ?

**Comment by thuongho** [comment]
> > Is this a good first issue?
> 
> I think it would be! What do you think @thuongho ?

Yes I agree, it might be a good first issue. Not too complicated, but at the same time has a bit of trickiness (possibly from integrating the side menu built for faq) that will give a rewarding challenge.

**Comment by kengoy** [comment]
Assign @devdav1999 to this.
